services/app-web/src/components/Form/FieldYesNo/FieldYesNo.tsx

```diff
@@ -105,3 +88,24 @@ export const FieldYesNo = ({
         </Fieldset>
     )
 }
+
+// HELPERS
+export type FieldYesNoFormValue = 'YES' | 'NO' | undefined // Use for formik string value
```

Comment:
I feel like I’ve tried and failed before but it would be nice if we could figure out how to make these fields store true/false instead of YES NO in formik just to cut down on the different representations here. 

Would it make sense to use a map here to make it really explicit the relationship between these? Or would that just make this more complicated?

`const fieldValueMapping: { true: { userValue: ‘Yes’, formikValue: ‘YES’}, false: … undefined: …}`

Reply:
I like the idea of making things even more explicit.

The issues with the dictionary I see are that booleans as keys won't work and then we still have to handle fallbacks. The functions currently all have a fallback or default value to use if nothing matches. I don't think you could use the dictionary directly for those reasons _(not sure if that's what you were intending though, did I understand correctly?)_. 

Another random idea: a single exported function that takes any valid values for this field something like `getYesNoValue(value, type) => value `. 

Value is `FieldYesNoUserValue | FieldYesFormValue |  FieldYesNoBooleanValue`
Type is some new enum we make to represent each possible type.

Does that seem better?